_posts/2022-11-12-jekyll-blog-tutorial.md

```diff
@@ -0,0 +1,76 @@
+ 
+## Resources
+- Lauren's slides:  [https://docs.google.com/presentation/d/1V3GeePgAsaDH3CP10u4Kzz4xn0_7komA/edit#slide=id.p1](https://docs.google.com/presentation/d/1V3GeePgAsaDH3CP10u4Kzz4xn0_7komA/edit#slide=id.p1)
+- IWD 1-minute video: [https://youtu.be/Z81fx_Fa34o](https://www.youtube.com/watch?v=Z81fx_Fa34o)
+- scikit-learn blog:  [https://blog.scikit-learn.org](https://blog.scikit-learn.org)
+- Reshama's blog:  [https://reshamas.github.io](https://reshamas.github.io)
+- Lauren's blog: [https://laurburke.github.io](https://laurburke.github.io)
+
+
+
+## Section Timestamps of Video  
+- [00:00:00](https://www.youtube.com/watch?v=7SBXl94xNl8&t=0s) Data Umbrella introduction by Reshama
+- [00:03:50](https://www.youtube.com/watch?v=7SBXl94xNl8&t=230s) Reshama introduces Lauren
+- [00:05:15](https://www.youtube.com/watch?v=7SBXl94xNl8&t=315s) Jekyll blog Presentation by Lauren
+- [00:05:48](https://www.youtube.com/watch?v=7SBXl94xNl8&t=348s) Intro & Agenda
+- [00:06:12](https://www.youtube.com/watch?v=7SBXl94xNl8&t=372s) Section 1: Background
+- [00:06:15](https://www.youtube.com/watch?v=7SBXl94xNl8&t=375s) Why create a personal website/blog?
+- [00:07:24](https://www.youtube.com/watch?v=7SBXl94xNl8&t=444s) What are Jekyll & GitHub Pages?
+- [00:08:05](https://www.youtube.com/watch?v=7SBXl94xNl8&t=485s) Why use Jekyll / GitHub Pages?
+- [00:08:50](https://www.youtube.com/watch?v=7SBXl94xNl8&t=530s) Section 2: Creating Your Site
+- [00:08:53](https://www.youtube.com/watch?v=7SBXl94xNl8&t=533s) Prerequisites
+- [00:09:30](https://www.youtube.com/watch?v=7SBXl94xNl8&t=570s) Install Jekyll
+- [00:10:05](https://www.youtube.com/watch?v=7SBXl94xNl8&t=605s) Create a New Jekyll Project
+- [00:10:45](https://www.youtube.com/watch?v=7SBXl94xNl8&t=645s) Run your site locally
+- [00:11:16](https://www.youtube.com/watch?v=7SBXl94xNl8&t=676s) What's inside the project?
+- [00:12:44](https://www.youtube.com/watch?v=7SBXl94xNl8&t=764s) Updating Site Configuration
+- [00:13:32](https://www.youtube.com/watch?v=7SBXl94xNl8&t=812s) Section 3: Hosting on GitHub Pages
+- [00:13:39](https://www.youtube.com/watch?v=7SBXl94xNl8&t=819s) Initialize the Repository
+- [00:14:15](https://www.youtube.com/watch?v=7SBXl94xNl8&t=855s) Add the GitHub Pages Gem
+- [00:14:48](https://www.youtube.com/watch?v=7SBXl94xNl8&t=888s) Personal Access Tokens (PAT)
+- [00:16:16](https://www.youtube.com/watch?v=7SBXl94xNl8&t=976s) Push to GitHub
+- [00:18:05](https://www.youtube.com/watch?v=7SBXl94xNl8&t=1085s) Section 4: Customization Options
+- [00:18:13](https://www.youtube.com/watch?v=7SBXl94xNl8&t=1093s) 1. Theme
+- [00:21:04](https://www.youtube.com/watch?v=7SBXl94xNl8&t=1264s) 2. Posts & Pages
+- [00:28:46](https://www.youtube.com/watch?v=7SBXl94xNl8&t=1726s) 3. Blog Settings
+- [00:32:33](https://www.youtube.com/watch?v=7SBXl94xNl8&t=1953s) 4. Other Customizations
+- [00:34:46](https://www.youtube.com/watch?v=7SBXl94xNl8&t=2086s) -- Q & A --
+
```

Comment:
```suggestion
```suggestion
## About the Speaker
### Bio
Lauren Burke is a Data Scientist in the healthcare technology space. She began her data science career in the retail space, where she developed solutions across several areas, including supply chain, inventory management, and enterprise business. She recently transitioned into product data science after spending some time implementing forecasting solutions to support new and renewal business opportunities. Lauren is an active member of her local and larger tech communities. She is the Director of Operations of Women in Analytics, a member of TECH CORPS' Regional Steering Committee, Vice-Chair of COSI's Kinetic Emerging Leaders Board, and a member of the scikit-learn Communication Team.

```
```